promote/conda_to_conda.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
```

Comment:
I know bash script style is a very... ahem... personal matter. But I would suggest that we run `-x` on our scripts in CI (and in general, avoid suppressing output, e.g., piping to `/dev/null`). My personal reasoning for this, is at some point in the future I'm going to be reading the tea leaves for some build that failed in the past that I cannot conveniently rerun, and I want enough information to figure out what went wrong without having to go through another cycle of pushing and seeing what happened. The more information that's printed in the logs, the better for working out the cause of the problem post facto. These logs aren't going to be read by uninformed users, let's put as much info in them, and accept that maybe you have to scroll through some noise when diagnosing them.